scale/cli/management/commands/migratedata.py

```diff
@@ -118,8 +119,10 @@ def handle(self, *args, **options):
                     ingest.ingest_ended = timezone.now()
                     ingest.source_file = sf
                     ingest.save()
-                    if options['recipe']:
-                        RecipeTriggerHandler().process_ingested_source_file(ingest.id, ingest.source_file, ingest.ingest_ended)
+                    if options['recipe_type']:
+                        IngestRecipeHandler().process_ingested_source_file(ingest.id, ingest.source_file, ingest.ingest_ended, options['recipe_type'])
+                    else:
+                        IngestTriggerHandler().process_ingested_source_file(ingest.source_file, ingest.ingest_ended)
```

Comment:
IngestTriggerHandler was removed in v6. Only IngestRecipeHandler should be used.